### DISCUSSION TITLE
Is there a way to leave out series from the summed amount?

### DISCUSSION THREAD

**Discussion by radonyizsolt** [apexcharts/apexcharts.js#4190] [Q&A]
Hi!
I am trying to make a trendline in a stacked bar chart, but my Y axis max value shows the all the series are counted. Is there a way to exclude one series from this so my Yaxis max value can be a value based on the first two series?

Examle:

> series = [
                    {
                        name: "Series1",
                        type: "bar",
                        data: [
                            2.5, 1.2, 1, 1.5, 1.1, 5, 2, 1.8, 1.8, 1, 1, 0.5,
                        ],
                    },
                    {
                        name: "Series2",
                        type: "bar",
                        data: [1.5, 1.1, 5, 2, 1, 1.8, 1, 1, 0.5, 2.5, 1.2, 1],
                    },
                    {
                        name: "Series3",
                        type: "line",
                        data: [1.5, 1.1, 5, 2, 1, 1.8, 1, 1, 0.5, 2.5, 1.2, 1],
                    },
                ]

Thank you!

**Comment by rosco54** [comment]
Your example, with the added minimal chart: {} spec below, stacks series1 and series2 (type "bar") and draws the line separately. I get series 3 stacked if the chart type is changed to "bar" instead of "line".

        series: [
        {
        name: "Series1",
        type: "bar",
        data: [2.5, 1.2, 1, 1.5, 1.1, 5, 2, 1.8, 1.8, 1, 1, 0.5],
        },
        {
        name: "Series2",
        type: "bar",
        data: [1.5, 1.1, 5, 2, 1, 1.8, 1, 1, 0.5, 2.5, 1.2, 1],
        },
        {
        name: "Series3",
        type: "line",
        data: [1.5, 1.1, 5, 2, 1, 1.8, 1, 1, 0.5, 2.5, 1.2, 1],
        },
        ],
      chart: {
        type: 'line',
        stacked: true
      }
